tianshou/highlevel/config.py

```diff
@@ -116,14 +126,22 @@ class SamplingConfig(ToStringMixin):
     """
 
     replay_buffer_ignore_obs_next: bool = False
+    """whether to ignore the `obs_next` field in the collected samples when storing them in the
+    buffer and instead use the one-in-the-future of `obs` as the next observation.
+    This can be useful for very large observations, like for Atari, in order to save RAM.
+
+    However, setting this to True **may introduce an error** at the last steps of episodes! Should
+    only be used in exceptional cases and only when you know what you are doing.
+    Currently only used in Atari examples and may be removed in the future!
+    """
 
     replay_buffer_save_only_last_obs: bool = False
-    """if True, for the case where the environment outputs stacked frames (e.g. because it
-    is using a `FrameStack` wrapper), save only the most recent frame so as not to duplicate
-    observations in buffer memory. Specifically, if the environment outputs observations `obs` with
-    shape (N, ...), only obs[-1] of shape (...) will be stored.
-    Frame stacking with a fixed number of frames can then be recreated at the buffer level by setting
-    :attr:`replay_buffer_stack_num`.
```

Comment:
The old docstring was better.

Reply:
I didn't do that consciously, probably a late-night copy paste error that escaped my attention. Will revert

Reply:
Done